server/src/db/dal/Initialized-data.ts

```diff
@@ -0,0 +1,127 @@
+},{
+    name: 'אוכל',
+    sentenceBeginning: 'אני רוצה ',
+    imageUrl: 'https://i.ibb.co/qCKxqfs/food.jpg',
+    itemNames: ['פיצה', 'המבורגר', 'קולה', 'בננה', 'גלידה', 'שוקולד', 'מלפפון', 'עגבניה', 'פלפל', 'אננס', 'תות', 'אפרסק', 'כריך'
+    , 'חביתה', 'ביצה', 'קציצה', 'פסטה', 'אורז', 'פירה', 'פתיתים', 'סושי', 'נודלס', 'מים', 'מיץ', 'סודה', 'שניצל', 'בשר', 'קורנפלקס'
+    , 'לחם', 'לחמניה', 'שוקו', 'חלב', 'שקשוקה', 'חומוס', 'סלט', 'גזר', 'תפוח', 'אגס', 'קינוח', 'עוגה', 'עוגיה', 'גבינה', 'טוסט']
+},{
+    name: 'משחק',
+    sentenceBeginning: 'אני רוצה לשחק ב',
+    imageUrl: 'https://i.ibb.co/PNfDnJM/games.jpg',
+    itemNames: ['כדורסל', 'כדורגל', 'לגו', 'פאזל', 'מחשב', 'בובות', 'קוביות', 'משחק קופסה', 'כדור', 'חברים', 'חול']
+},{
+    name: 'היגיינה',
+    sentenceBeginning: 'אני רוצה ',
+    imageUrl: 'https://i.ibb.co/Y7NQ03j/hygiene.jpg',
+    itemNames: ['לנקות', 'להתקלח', 'לשטוף ידיים', 'לצחצח שיניים', 'שירותים']
+},{
+    name: 'ספורט',
+    sentenceBeginning: 'אני רוצה ',
+    imageUrl: 'https://i.ibb.co/SK6rnBQ/sport.jpg',
+    itemNames: ['לשחק ', 'לשחק כדורגל', 'לשחק כדורסל', 'לרוץ', 'לקפוץ', 'להתעמל', 'לשחק כדורעף', 'לרקוד', 'לשחות', 'לרכב על אופניים']
+},{
+    name: 'משפחה וחברים',
+    sentenceBeginning: '',
+    imageUrl: 'https://i.ibb.co/Gn7QZfr/family.jpg',
+    itemNames:['סבתא','סבא', 'אחות', 'אח', 'אבא', 'אמא', 'חֲבֵרָה', 'חבר', 'דּוֹד', 'דודה', 'בן דּוֹד', 'בת דודה', 'אחיין', 'אחיינית']
+},{
+    name: 'מקומות',
+    sentenceBeginning: 'אני רוצה ללכת ל',
+    imageUrl: 'https://i.ibb.co/XLdy7rp/locations.jpg',
+    itemNames: ['בית', 'בית ספר', 'יער', 'חוג', 'משחקיה', 'קניון', 'מגרש', 'חברים', 'כיתה', 'קייטנה', 'סבא וסבתא', 'בריכה'
+    , 'ים', 'גן חיות', 'פארק']
+},{
+    name: 'לימודים',
+    sentenceBeginning: 'אני רוצה ',
+    imageUrl: 'https://i.ibb.co/d2qNZXP/study.jpg',
+    itemNames: ['מחק', 'עפרון', 'ספר', 'מספריים', 'מחברת', 'מורה', 'שיעור', 'הפסקה', 'מחדד', 'עט', 'צבעים', 'קלמר', 'תיק']
+},{
+    name: 'צפייה',
+    sentenceBeginning: 'אני רוצה לראות ',
+    imageUrl: 'https://i.ibb.co/8cbj9VF/watch.jpg',
+    itemNames: ['טלויזיה', 'סרט', 'סדרה', 'הופעה', 'כוכבים']
+},{
+    name: 'שיחה',
+    sentenceBeginning: '',
+    imageUrl: 'https://i.ibb.co/x6KsbCF/chat.jpg',
+    itemNames: ['שלום', 'מה שלומך?', 'אני', 'אתה', 'את', 'אנחנו', 'רוצה?', 'יודע?', 'איפה?', 'מתי?', 'כמה?', 'למה?', 'מה?', 'להתראות']
+},{
+    name: 'פעולות',
+    sentenceBeginning: 'אני רוצה ',
+    imageUrl: 'https://i.ibb.co/x6KsbCF/chat.jpg',
+    itemNames: ['ללכת לישון', 'לשחק', 'לנקות', 'ללכת', 'לטייל', 'לצייר', 'לנוח', 'לבשל', 'לשיר', 'לכתוב', 'לקנות']
+}];
+
+export const hardcodedItems : HardcodedItem[] = [
+{name: 'איפה?', imageUrl: 'https://images.unsplash.com/photo-1535745122259-f1e187953c4c?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=M3w0NDA2MTh8MHwxfHNlYXJjaHw0fHwlRDclOTAlRDclOTklRDclQTQlRDclOTR8aGV8MXx8fHwxNjg0Nzc0MjgzfDA&ixlib=rb-4.0.3&q=80&w=1080'},
```

Comment:
```
export const hardcodedImageUrls: Record<string, string> = {
    'איפה?': "https://images.unsplash......",
    'אנחנו': "https://images.unsplash......"
}
```

Then access it in O(1). Looks nicer too instead of two fields